lib/v2/picuki/profile.js

```diff
@@ -0,0 +1,96 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const chrono = require('chrono-node');
+const { art } = require('@/utils/render');
+const path = require('path');
+
+module.exports = async (ctx) => {
+    const id = ctx.params.id;
+    const displayVideo = ctx.params.displayVideo !== '0';
+
+    const profileUrl = `https://www.picuki.com/profile/${id}`;
+
+    const response = await got.get(profileUrl, {
+        headers: {
+            'User-Agent': 'Mozilla/5.0 (iPhone; CPU iPhone OS 11_0 like Mac OS X) AppleWebKit/604.1.38 (KHTML, like Gecko) Version/11.0 Mobile/15A372 Safari/604.1',
```

Comment:
Any reason why iPhone's UA is used? RSSHub's default UA seems also working.

Reply:
No particular reason. It is just an attempt to avoid being rate limited. Currently, I have no idea about this will help or not. But I consider that it won't make things worse. If you are still more willing to use RSSHub's default UA, that's fine with me.

Reply:
RSSHub(got)'s UA is defined here.
https://github.com/DIYgod/RSSHub/blob/cdcb2e580bd9989f1a57574aa48be2c2693a006d/lib/config.js#L40
https://github.com/DIYgod/RSSHub/blob/cdcb2e580bd9989f1a57574aa48be2c2693a006d/lib/utils/got.js#L34

It can be changed via env.